pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -53,6 +54,10 @@ import (
 	"sigs.k8s.io/kueue/pkg/workloadslicing"
 )
 
+const (
+	WorkloadEvictedOnManagerCluster = "EvictedOnManagerCluster"
```

Comment:
Could you move this reason to API: https://github.com/kubernetes-sigs/kueue/blob/5233f1b3204f293383f347e643bb113611fbccc3/apis/kueue/v1beta2/workload_types.go#L905?

This is user-facing, and this should be managed as part of the API.